### DISCUSSION TITLE
Unable to add a class name to a custom property

### DISCUSSION THREAD

**Discussion by jipika** [tailwindlabs/tailwindcss#9551] [Help] [answered]
![image](https://user-images.githubusercontent.com/59040157/195793554-63bb8c28-f3d1-4a3d-b209-cef45b6ca0f8.png)

I've configured this class name in vscode but it's not generating correctly

**Comment by thecrypticace** [comment]
Hey, can you provide more details about what you are seeing, how you are using Tailwind CSS, etc…?
First thought: Have you added your vue files to your content paths?

This alone isn't enough information to go on so any additional info you can provide would be helpful.

**Reply by jipika** [reply]
![image](https://user-images.githubusercontent.com/59040157/196071747-6637cbcc-d4f7-474d-8cae-a41bbc7d729b.png)

All others that have been added are generated correctly only headerClass cannot be generated

**Reply by jipika** [reply]
This property is styled by the prop modification inside my component

**Comment by Carlsia** [comment]
A commitment of appreciation is all together for sharing, I found a tremendous store of stimulating information here. A striking post, incredibly grateful and obliging that you will make on a very major level more posts like this one. [BayCare Patient Portal App](https://www.baycarepatientportal.net/)

**Comment by jipika** [comment, marked as answer]
He generated it correctly, but he didn't have enough weights
